firefox_puppeteer/ui/windows.py

```diff
@@ -413,6 +415,11 @@ def __init__(self, *args, **kwargs):
         self._tabbar = None
 
     @property
+    def default_homepage(self):
+        """Returns the value of the preference browser.startup.homepage."""
```

Comment:
Just say `The default homepage as used by the current locale`. Also please add the `:returns` line.